src/state/user/hooks.tsx

```diff
@@ -101,6 +103,7 @@ export function useExpertModeManager(): [boolean, () => void] {
 
   const toggleSetExpertMode = useCallback(() => {
     dispatch(updateUserExpertMode({ userExpertMode: !expertMode }))
+    user.set(CUSTOM_USER_PROPERTIES.EXPERT_MODE, !expertMode)
```

Comment:
if you're gonna do this in the effect above, then don't need this.

Reply:
ah yep youre right removed thanks.